lib/msf/core/post/common.rb

```diff
@@ -198,6 +198,13 @@ def cmd_exec(cmd, args=nil, time_out=15)
       end
 
       process.close
+    when /powershell/
```

Comment:
I'm tempted or deleting it :) and just do session.shell_command from shell_to_meterpreter